service/history/execution/context_util.go

```diff
@@ -155,5 +155,15 @@ func emitWorkflowCompletionStats(
 			tag.WorkflowRunID(runID),
 			tag.WorkflowDomainName(domainName),
 		)
+	case types.EventTypeWorkflowExecutionContinuedAsNew:
+		scope.IncCounter(metrics.WorkflowContinuedAsNew)
+	default:
+		scope.IncCounter(metrics.WorkflowCompletedUnknownType)
+		logger.Info("NOT emitting completion stats for event type",
```

Comment:
Probably we should make this a `.Warn` and messaging around not knowing the completion event could be better:
`Workflow completed with an unknown even type`

Reply:
+1